src/WebJobs.Extensions.Http/Config/HttpOptions.cs

```diff
@@ -52,6 +52,12 @@ public HttpOptions()
         /// </summary>
         public bool DynamicThrottlesEnabled { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether chunked 
+        /// transfer should be enabled.
+        /// </summary>
+        public bool ChunkedTransferEnabled { get; set; }
```

Comment:
This needs a name that better describes what it does. This doesn't enable chunked transfers, it enables SDK binding to body properties on chunked requests.

Reply:
Something like `EnableChunkedRequestBinding`